render.js

```diff
@@ -0,0 +1,361 @@
+import { getCompleteData, getFromDB, getIncompleteData } from "./db_calls.js"
+import { addTodo, calculateElapsedTime, completeTodo, deleteTodo, editTodo, saveTodo } from "./event_handlers.js"
+
+const limit = 6; // number of cards to be displayed without pressing "Load More"
```

Comment:
We can use meaningful variable names like this and remove the necessity of commenting here bhai
```suggestion
const numberOfCardsToShow = 6;
```